aten/src/ATen/detail/CUDAHooksInterface.h

```diff
@@ -23,7 +23,8 @@ namespace at {
 constexpr const char* CUDA_HELP =
   "PyTorch splits its backend into two shared libraries: a CPU library "
   "and a CUDA library; this error has occurred because you are trying "
-  "to use some CUDA functionality, but the CUDA library has not been "
+  "to use some CUDA functionality, but the CUDA library does not exist."
```

Comment:
This change isn't needed anymore, I'll revert.

Initially, this PR made it so that this was being hit in some cases when pytorch was being built without CUDA, and I wanted to include that possibility in the error message. But the most recent changes make it so it won't be hit anymore.